frontend/package.json

```diff
@@ -31,9 +32,11 @@
     "@storybook/vue3": "^7.0.7",
     "@storybook/vue3-vite": "^7.0.7",
     "@types/jsdom": "^21.1.0",
-    "@types/node": "^18.14.2",
+    "@types/node": "^18.16.3",
     "@vitejs/plugin-vue": "^4.0.0",
     "@vitejs/plugin-vue-jsx": "^3.0.0",
+    "@vue/cli-plugin-typescript": "^5.0.8",
+    "@vue/cli-service": "^5.0.8",
```

Comment:
What are these libraries for, are they required?

Reply:
deleted unnecessary libraries and fixed Eslint issues